scripts/world_bank/worldbank.py

```diff
@@ -0,0 +1,268 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Fetches, cleans, outputs TMCFs and CSVs for all WorldBank development indicator codes provided in WorldBankIndicators.csv
+for all years for all countries provided in WorldBankCountries.csv """
+
+from absl import app
+import pandas as pd
+import numpy as np
+import itertools
+import requests, zipfile, io
+import re
+
+WORLDBANK_COL_REMAP = {
+  'Country Name':'CountryName',
+  'Country Code':'CountryCode',
+  'Indicator Name': 'IndicatorName',
+  'Indicator Code': 'IndicatorCode'
+}
+
+# The maximum number of constraints in the indicators to generate
+MAX_CONSTRAINTS = 3
+
+TEMPLATE_TMCF = """
+Node: E:BLSJolts->E0
+typeOf: dcs:StatVarObservation
+variableMeasured: C:WorldBank->StatisticalVariable
+observationDate: C:WorldBank->Year
+observationPeriod: P1Y
+observationAbout: C:WorldBank->CountryNode
+value: C:WorldBank->Value
+"""
+
+TEMPLATE_STAT_VAR = """
+Node: dcid:WorldBank/{INDICATOR}
+name: "{NAME}"
+description: "{DESCRIPTION}"
+typeOf: dcs:StatisticalVariable
+populationType: {populationType}
+statType: dcs:{statType}
+measuredProperty: dcs:{measuredProperty}
+{CONSTRAINTS}
+"""
+
+COUNTRY_MCF = """
+Node: E:WorldBank->E{X}
+typeOf: dcs:Country
+isoCode: dcs:{ISO_CODE}
+"""
+
+def read_worldbank(country_iso):
+  """ Fetches and tidies all ~1500 Worldbank indicators for a given 3 digit ISO code.
```

Comment:
http://google.github.io/styleguide/pyguide.html#32-line-length

Most editors can be configured to wrap lines automatically.  If you want help, please specify your editor.